### PR TITLE
PIM-3662 Handle product template media after saving variant group

### PR DESCRIPTION
**Pull request by filipsalpe**
| Q                    | A
| -------------------- | ---
| Bug fix?             | N
| New feature?         | Y
| BC breaks?           | N
| CI currently passes? | Y
| Tests pass?          | Y
| Scenarios pass?      | ?
| Checkstyle issues?*  | N
| PMD issues?**        | N
| Changelog updated?   | 
| Fixed tickets        |
| DB schema updated?   |
| Migration script?    |
| Doc PR               |

### CODE REVIEW

File: src/Pim/Bundle/CatalogBundle/Saver/GroupSaver.php [hunk 1]
```diff
@@ -77,6 +84,13 @@ public function save($group, array $options = [])
             $this->removeProducts($options['remove_products']);
         }
 
+        if ($group->getType()->isVariant()) {
+            $template = $group->getProductTemplate();
+            if (null !== $template) {
+                $this->templateMediaManager->handleProductTemplateMedia($template);
```
Comment (nidup) [inline src/Pim/Bundle/CatalogBundle/Saver/GroupSaver.php]: would put this in ProductTemplateApplier, no ?

Comment (filipsalpe) [inline src/Pim/Bundle/CatalogBundle/Saver/GroupSaver.php]: Nope, this is just for the group template itself

File: src/Pim/Bundle/CatalogBundle/Resources/config/managers.yml [hunk 2]
```diff
@@ -16,6 +16,7 @@ parameters:
     pim_catalog.manager.product_mass_action.class: Pim\Bundle\CatalogBundle\Manager\ProductMassActionManager
     pim_catalog.manager.product_category.class:    Pim\Bundle\CatalogBundle\Manager\ProductCategoryManager
     pim_catalog.manager.product_template_attributes.class: Pim\Bundle\CatalogBundle\Manager\ProductTemplateAttributesManager
+    pim_catalog.manager.product_template_media.class:      Pim\Bundle\CatalogBundle\Manager\ProductTemplateMediaManager
```
Comment (sumbobyboys) [inline src/Pim/Bundle/CatalogBundle/Resources/config/managers.yml]: not aligned. Go to hell

File: src/Pim/Bundle/CatalogBundle/Saver/GroupSaver.php (hunk 1 above)
Comment (nidup) [inline src/Pim/Bundle/CatalogBundle/Saver/GroupSaver.php]: you right

**Pull request merged**